### ISSUE TITLE
Option to prefix entity names with car name

### ISSUE THREAD

**Issue by CyberDNS** [wubbl0rz/FiatChamp#31]
Hi @wubbl0rz,

Thank you very much for this addon, the times using the iobroker integration will end soon.😆

The possibility to prefix entity names with e.g. the car name, will help when you have multiple cars or even if you have only one car it makes the entities easier to find in home assistant, especially those with more generic names like button.blink etc.
Also this option will regroup the entities under the set prefix/car name.

Here an example:
button.blink (without the option set) => button.myhappycar_blink (with the option set)

I will provide a pull request in the next days with a code suggestion if this change is appreciated.

Best regards,
CyberDNS

**Comment by wubbl0rz** [comment]
have you tried to give your car a "nickname" in uconnect ?  https://github.com/wubbl0rz/FiatChamp/blob/master/FiatChampAddon/FiatClient/Program.cs#L90  ... my car has no nickname so i never checked if its really working :thinking:

**Comment by CyberDNS** [comment]
Yes, my car has a nickname and in the code before the change, the nickname is correctly read into HaDevice name attribute.

**Comment by AirBeeSee** [comment]
I look forward to this improvement
